test/test-ipc.c

```diff
@@ -203,7 +203,7 @@ static void on_read(uv_stream_t* handle,
     /* Make sure that the expected data is correctly multiplexed. */
     ASSERT_MEM_EQ("hello\n", buf->base, nread);
 
-    outbuf = uv_buf_init("world\n", 6);
+    outbuf = uv_buf_init("foobar\n", 7);
```

Comment:
Why this change?

Reply:
I changed this because I wanted the read and writes to be different lengths ("hello" and "world" were previously the same number of bytes)